spark/v3.4/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMergeOnReadUpdate.java

```diff
@@ -160,7 +159,7 @@ public void testUnpartitionedPositionDeletesAreMaintainedDuringUpdate() {
         sql("SELECT * FROM %s ORDER BY dep ASC, id ASC", selectTarget()));
   }
 
-  @TestTemplate
```

Comment:
this test wasn't executed earlier because the entire module wasn't set to running with JUnit5